skills/gitlab-mcp/reference/webhooks.md

```
@@ -11,6 +11,31 @@ list_webhooks
   project_id: "my-group/my-project"
 ```
 
+## Create / Update / Delete
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)